parking_lot/shr_sexualhealth.txt

```diff
@@ -10,80 +10,80 @@ CodeSystem:     UCUM = http://unitsofmeasure.org
 
 EntryElement:	PregnancyIntention
 Concept:		TBD
-Based on:		QuestionAnswerRecorded
+Based on:		SocialHistoryObservation
 Description:	"Whether the subject or his/her partner has a plan or desire to either become pregnant or have a child in the next year/near future."
 Value:			CodeableConcept from YesNoUnknownVS
-				TopicCode is LNC#64631-5
+				Code is LNC#64631-5
 
 EntryElement:	SexualIdentity
 Concept:		TBD
-Based on:		QuestionAnswerRecorded
+Based on:		SocialHistoryObservation
 Description:	"Information on gender identity and sexual orientation."
-				TopicCode is TBD#TBD
+				Code is TBD#TBD
 				PanelMembers.EvaluationTopic
 includes 0..1 	GenderIdentity
 includes 0..1	SexualOrientation
 
 	EntryElement:	GenderIdentity
 	Concept:		MTH#C0017249
-	Based on:		QuestionAnswerRecorded
+	Based on:		SocialHistoryObservation
 	Description:	"Subjective personal perception of one's own gender, which can be male, female, a blend of both, or neither."
 	Value:			CodeableConcept from GenderIdentityVS 
-					TopicCode is MTH#C0017249
+					Code is MTH#C0017249
 
 	EntryElement:	SexualOrientation
 	Concept:		MTH#C0205949
-	Based on:		QuestionAnswerRecorded
+	Based on:		SocialHistoryObservation
 	Description:	"An inherent and enduring emotional, romantic or sexual attraction to other people."
 	Value:			CodeableConcept from SexualOrientationVS
-					TopicCode is MTH#C0205949
+					Code is MTH#C0205949
 
 
 EntryElement:	SexualBehavior
 Concept:		MTH#C1314687
-Based on:		QuestionAnswerRecorded
+Based on:		SocialHistoryObservation
 Description:	"Experience with sexual intercourse."
 0..0			Value
-				TopicCode is MTH#C1314687
+				Code is MTH#C1314687
 				PanelMembers.EvaluationTopic 
 includes 1..*	SexualActivity
 includes 0..1	NumberOfSexualPartnersPastYear
 includes 0..*	IntercourseDifficulty
 
 	EntryElement:	SexualActivity
 	Concept:		MTH#C0241028
```

Comment:
MTH#C0241028 is "sexually active" which seems more like an assessment whether the person is having sex.
MTH#C0036864 is a better fit with a description of "Sex behavior" but also lists "sexual activity" as a synonym for the concept and also contains a richer set of relationships which further quality the type of sexual activity.